### PR TITLE
Updated translation for 'Bad credentials.' message

### PR DESCRIPTION
**Pull request by kozmenkoav**
Source string has been changed https://github.com/symfony/security-core/commit/f4e65fb1d5cca6057c3111784df733d0f04517f4

### CODE REVIEW

**Comment by kozmenkoav** [comment]
These updates successfully fixed the problem in my project based on sonata. But they can cause an issue with Symfony versions lower than 2.5. As an easiest solution we can add both versions of string to translation files.

**Comment by OskarStark** [comment]
:+1:  for both messages, but we should add a comment in the translation file for these strings

**Comment by kozmenkoav** [comment]
@OskarStark, please check. I've added comments and translation version with dots. It was challenging for Chinese, Japanese and Farsi

**Comment by OskarStark** [comment]
not possible to add a comment in farsi etc. ?

File: Resources/translations/FOSUserBundle.fa.yml [hunk 1]
```diff
@@ -14,6 +14,8 @@ group:
 # Security
 "Bad credentials": نام کاربری یا کلمه عبور غیر مجاز
 
+"Bad credentials.": نام کاربری یا کلمه عبور غیر مجاز.
```
Comment (OskarStark) [inline Resources/translations/FOSUserBundle.fa.yml]: i mean here

File: Resources/translations/FOSUserBundle.bg.yml [hunk 2]
```diff
@@ -14,6 +14,9 @@ group:
 # Security
 "Bad credentials": Невалидно потребителско име или парола
 
+# Since Symfony v2.6
```
Comment (OskarStark) [inline Resources/translations/FOSUserBundle.bg.yml]: i would use:

``Since Symfony 2.6 the error message has changed``

**Comment by OskarStark** [comment]
Fixes #1746

**Comment by kozmenkoav** [comment]
@OskarStark, please check my fixes.

**Comment by OskarStark** [comment]
For me this is RTM

Can you please rebase @kozmenkoav ?

**Comment by kozmenkoav** [comment]
@OskarStark, rebased

**Comment by XWB** [comment]
It still won't merge, can you recheck?

And perhaps you can re-open this PR against the 1.3 branch?

**Comment by kozmenkoav** [comment]
Please check new PR https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2042

**Pull request closed without merging**